app/modules/encounters/models.py

```diff
@@ -125,6 +125,16 @@ def add_annotation(self, annotation):
             with db.session.begin(subtransactions=True):
                 self.annotations.append(annotation)
 
+    def _merge_request_hash(self):
```

Comment:
This is not used in the encounter models code, only in the individuals models code, hence it should not be 'private', ie no preceding _

Reply:
done.